src/convertToObject.js

```diff
@@ -1,18 +1,16 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes string with styles
- * and returns object where CSS properties are keys
- * and values are the values of related CSS properties
-
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const newObj = {};
+  let property;
+  const newArr = sourceString.split(';\n');
```

Comment:
what if there is no \n in the source string? split by `;` better, you will always have it in css code

Reply:
Thanks, corrected that.